README.md

```diff
@@ -222,6 +222,14 @@ command not found: node
 
 # Getting Started
 
+The easiest way to install tea is with our installer:
+
+```sh
+sh <(curl https://tea.xyz)
```

Comment:
I actually deliberately removed this recently to encourage people to install via brew as our suggested workflow.

magic can then be added manually, and I intend to improve that.

there's too much worry regarding `curl | sh` installers nowadays and it hinders our adoption.

So yeah, please remove.

Reply:
So, would you like to leave only brew installation instruction in the README? I think it'd limit adoption further as well, especially for Linux folks, I guess brew is not that popular on that platform as it is on mac.

Reply:
The instructions also provide a one-liner for installation for people without brew.

so far we've experienced severe reluctance to use our curl installer. So severe I'm just removing it. Use brew or download the standalone `tea` binary.

It's weird since you install brew that way, and I wrote that curl installer too but people are what they are.

Reply:
edit: and you say the one-liner doesn't work for linux intel so we should fix that. I added a comment below about that.